### DISCUSSION TITLE
Can I change the name of the device?

### DISCUSSION THREAD

**Discussion by n1cklim** [FDH2/UxPlay#332] [Q&A] [answered]
I was wondering if I could change the name of the device. In this case, "UxPlay@UxPlay".

Thanks!

![image](https://github.com/user-attachments/assets/094ad239-a112-4a32-a12b-5698a283ac78)

**Comment by fduncanh** [comment, marked as answer]
NAME
       uxplay - start AirPlay server

SYNOPSIS
       uxplay [-n name] [-s wxh] [-p [n]] [more  OPTIONS  ...]

DESCRIPTION
       UxPlay 1.70: An open-source AirPlay mirroring (+ audio streaming) server:

OPTIONS
       -n name   Specify the network name of the AirPlay server

       -nh       Do NOT append "@hostname" at end of AirPlay server name

is this what you needed?

**Reply by n1cklim** [reply]
Yes, thank you!
